package.json

```diff
@@ -47,7 +47,7 @@
     "bl": "^2.2.1",
     "bson": "^1.1.4",
     "denque": "^1.4.1",
-    "require_optional": "^1.0.1",
+    "optional-require": "^1.0.2",
```

Comment:
https://github.com/mongodb/node-mongodb-native/blob/a83541b58fbeb9fc7e323d87734c53ca4d6d2755/package.json#L19-L25

This section of package.json was used by the old lib, we can drop this along with this change.

Reply:
Thanks, I've updated.